### DISCUSSION TITLE
Exclamation marks on tabs

### DISCUSSION THREAD

**Discussion by LAPIII** [WinMerge/winmerge#1230] [Q&A] [answered]
Do you know what I did to get a red exclamation mark on the left side of a tab? I continue to get these, do you know how I can stop it?

**Comment by sdottaka** [comment, marked as answer]
Do you mean the icon shown below? 

![image](https://user-images.githubusercontent.com/98126/156852900-45a0f813-f30f-4a8f-bb9e-1387d3266143.png)

If so, this icon indicates that the files are different. The only way to hide this icon is to revert to an older version of WinMerge (2.14.0 or earlier).
